netbox/netbox/models/features.py

```diff
@@ -118,6 +118,15 @@ def clone(self):
         if is_taggable(self):
             attrs['tags'] = [tag.pk for tag in self.tags.all()]
 
+        # check custom fields
+        if hasattr(self, 'custom_field_data'):
+            from extras.models import CustomField
+
+            for field in CustomField.objects.get_for_model(self):
```

Comment:
We should be able to use `get_custom_fields()` here.
```suggestion
            for field in model.get_custom_fields():
```